Applications/DataExplorer/DataView/ElementTreeModel.cpp

```diff
@@ -140,8 +140,8 @@ void ElementTreeModel::setMesh(MeshLib::Mesh const*const mesh)
 	_rootItem->appendChild(aabb_item);
 
 	const GeoLib::AABB<MeshLib::Node> aabb (MeshLib::MeshInformation::getBoundingBox(*mesh));
-	const MeshLib::Node& min = aabb.getMinPoint();
-	const MeshLib::Node& max = aabb.getMaxPoint();
+	auto min = aabb.getMinPoint();
```

Comment:
no need to write `auto&`? maybe there is no difference but `aabb.getMinPoint()` returns a reference.